astropy/config/tests/test_configs.py

```diff
@@ -99,67 +99,71 @@ def test_set_temp_cache_resets_on_exception(tmp_path):
+    testcfg = get_config("testpkg", rootname="astropy")
     parts = os.path.normpath(testcfg.filename).split(os.sep)
-    assert '.astropy' in parts or 'astropy' in parts
-    assert parts[-1] == 'testpkg.cfg'
-    configuration._cfgobjs['testpkg'] = None  # HACK
+    assert ".astropy" in parts or "astropy" in parts
+    assert parts[-1] == "testpkg.cfg"
+    configuration._cfgobjs["testpkg"] = None  # HACK
 
     # try with a different package name, no specified root name (should
     #   default to astropy):
-    testcfg = get_config('testpkg')
+    testcfg = get_config("testpkg")
     parts = os.path.normpath(testcfg.filename).split(os.sep)
-    assert '.astropy' in parts or 'astropy' in parts
-    assert parts[-1] == 'testpkg.cfg'
-    configuration._cfgobjs['testpkg'] = None  # HACK
+    assert ".astropy" in parts or "astropy" in parts
+    assert parts[-1] == "testpkg.cfg"
+    configuration._cfgobjs["testpkg"] = None  # HACK
 
     # try with a different package name, specified root name:
-    testcfg = get_config('testpkg', rootname='testpkg')
+    testcfg = get_config("testpkg", rootname="testpkg")
     parts = os.path.normpath(testcfg.filename).split(os.sep)
-    assert '.testpkg' in parts or 'testpkg' in parts
-    assert parts[-1] == 'testpkg.cfg'
-    configuration._cfgobjs['testpkg'] = None  # HACK
+    assert ".testpkg" in parts or "testpkg" in parts
+    assert parts[-1] == "testpkg.cfg"
+    configuration._cfgobjs["testpkg"] = None  # HACK
 
     # try with a subpackage with specified root name:
-    testcfg_sec = get_config('testpkg.somemodule', rootname='testpkg')
+    testcfg_sec = get_config("testpkg.somemodule", rootname="testpkg")
     parts = os.path.normpath(testcfg_sec.parent.filename).split(os.sep)
-    assert '.testpkg' in parts or 'testpkg' in parts
-    assert parts[-1] == 'testpkg.cfg'
-    configuration._cfgobjs['testpkg'] = None  # HACK
+    assert ".testpkg" in parts or "testpkg" in parts
+    assert parts[-1] == "testpkg.cfg"
+    configuration._cfgobjs["testpkg"] = None  # HACK
 
-    reload_config('astropy')
+    reload_config("astropy")
 
 
 def check_config(conf):
     # test that the output contains some lines that we expect
-    assert '# unicode_output = False' in conf
-    assert '[io.fits]' in conf
-    assert '[table]' in conf
-    assert '# replace_warnings = ,' in conf
-    assert '[table.jsviewer]' in conf
-    assert '# css_urls = https://cdn.datatables.net/1.10.12/css/jquery.dataTables.css,' in conf
-    assert '[visualization.wcsaxes]' in conf
-    assert '## Whether to log exceptions before raising them.' in conf
-    assert '# log_exceptions = False' in conf
+    assert "# unicode_output = False" in conf
+    assert "[io.fits]" in conf
+    assert "[table]" in conf
+    assert "# replace_warnings = ," in conf
+    assert "[table.jsviewer]" in conf
+    assert (
+        "# css_urls = https://cdn.datatables.net/1.10.12/css/jquery.dataTables.css,"
+        in conf
+    )
```

Comment:
I found this particular case *very* hard to understand/read with the multi-line form. Are we willing to skip this one (@nstarman , if you strongly disagree with me I'm ok with backing down)
```suggestion
    assert "# css_urls = https://cdn.datatables.net/1.10.12/css/jquery.dataTables.css," in conf  # fmt: skip
```